query10.sql

```diff
@@ -4,4 +4,9 @@
 */
 
 -- Enter your SQL query here
-select ...
+SELECT indego_station_statuses.id,
+		indego_station_statuses.name,
+      st_transform(st_setsrid(the_geom, 4326), 3857) as the_geom_webmercator,
+      st_distance(st_transform(st_setsrid(the_geom, 4326), 3857), st_transform(st_setsrid(st_makepoint(-75.1926 ,39.9522), 4326), 3857)
```

Comment:
Remember that SRID 3857 is going to be pretty distorted at latitudes far from the equator. Even though the order below is correct (because everything around this latitude is distorted _the same amount_, so the distances are correct _relative to each other_), the actual _absolute_ distances are over-inflated. Either SRID [2272](https://spatialreference.org/ref/epsg/2272/) or [32129](https://spatialreference.org/ref/epsg/32129/) would be better for Southern PA.